app/controllers/recipes_ingredients_controller.rb

```diff
@@ -0,0 +1,3 @@
+class RecipesIngredientsController < ApplicationController
+
```

Comment:
Yes this is it! Usually, per convention, we name joins controllers with a plural only on the second word - so RecipeIngredients with recipe as singular.

Reply:
Nice work playing around with this extension!! Even though it's not the most RESTful implementation, it's still something to celebrate 🥳